spring-batch-docs/asciidoc/readersAndWriters.adoc

```diff
@@ -88,7 +89,7 @@ generally expected that items are 'batched' together into a chunk and then outpu
 interface accepts a list of items, rather than an item by itself. After writing out the
 list, any flushing that may be necessary can be performed before returning from the write
 method. For example, if writing to a Hibernate DAO, multiple calls to write can be made,
-one for each item. The writer can then call `close` on the hibernate session before
```

Comment:
The `HibernateItemWriter` calls `flush` in the write method. The session can be used for multiple write calls.